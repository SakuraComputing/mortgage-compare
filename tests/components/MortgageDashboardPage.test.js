import React from 'react';
import { shallow, configure } from 'enzyme';
import MortgageDasboardPage from '../../app/components/MortgageDashboardPage';

describe('Mortagage Dashboard Page', () => {
  it('should render the page correctly', () => {
    const wrapper = shallow(<MortgageDasboardPage />);
    expect(wrapper).toMatchSnapshot();
  })
});

