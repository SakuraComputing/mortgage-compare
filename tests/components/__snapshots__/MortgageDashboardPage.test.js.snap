// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mortagage Dashboard Page should render the page correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MortgageDasboardPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <MortgageContainer
        quotes={
          Array [
            Object {
              "fixed": 2,
              "id": 1,
              "initialContribution": 25000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 251000,
              "provider": "santander",
              "term": 14,
            },
            Object {
              "fixed": 5,
              "id": 2,
              "initialContribution": 25000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 251000,
              "provider": "halifax",
              "term": 14,
            },
            Object {
              "fixed": 5,
              "id": 3,
              "initialContribution": 26000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 251000,
              "provider": "halifax",
              "term": 14,
            },
            Object {
              "fixed": 2,
              "id": 4,
              "initialContribution": 27000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 257000,
              "provider": "santander",
              "term": 14,
            },
          ]
        }
      />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "quotes": Array [
          Object {
            "fixed": 2,
            "id": 1,
            "initialContribution": 25000,
            "monthlyPayment": 1976,
            "mortgageAmount": 233800,
            "mortgageTotal": 251000,
            "provider": "santander",
            "term": 14,
          },
          Object {
            "fixed": 5,
            "id": 2,
            "initialContribution": 25000,
            "monthlyPayment": 1976,
            "mortgageAmount": 233800,
            "mortgageTotal": 251000,
            "provider": "halifax",
            "term": 14,
          },
          Object {
            "fixed": 5,
            "id": 3,
            "initialContribution": 26000,
            "monthlyPayment": 1976,
            "mortgageAmount": 233800,
            "mortgageTotal": 251000,
            "provider": "halifax",
            "term": 14,
          },
          Object {
            "fixed": 2,
            "id": 4,
            "initialContribution": 27000,
            "monthlyPayment": 1976,
            "mortgageAmount": 233800,
            "mortgageTotal": 257000,
            "provider": "santander",
            "term": 14,
          },
        ],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <MortgageContainer
          quotes={
            Array [
              Object {
                "fixed": 2,
                "id": 1,
                "initialContribution": 25000,
                "monthlyPayment": 1976,
                "mortgageAmount": 233800,
                "mortgageTotal": 251000,
                "provider": "santander",
                "term": 14,
              },
              Object {
                "fixed": 5,
                "id": 2,
                "initialContribution": 25000,
                "monthlyPayment": 1976,
                "mortgageAmount": 233800,
                "mortgageTotal": 251000,
                "provider": "halifax",
                "term": 14,
              },
              Object {
                "fixed": 5,
                "id": 3,
                "initialContribution": 26000,
                "monthlyPayment": 1976,
                "mortgageAmount": 233800,
                "mortgageTotal": 251000,
                "provider": "halifax",
                "term": 14,
              },
              Object {
                "fixed": 2,
                "id": 4,
                "initialContribution": 27000,
                "monthlyPayment": 1976,
                "mortgageAmount": 233800,
                "mortgageTotal": 257000,
                "provider": "santander",
                "term": 14,
              },
            ]
          }
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "quotes": Array [
            Object {
              "fixed": 2,
              "id": 1,
              "initialContribution": 25000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 251000,
              "provider": "santander",
              "term": 14,
            },
            Object {
              "fixed": 5,
              "id": 2,
              "initialContribution": 25000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 251000,
              "provider": "halifax",
              "term": 14,
            },
            Object {
              "fixed": 5,
              "id": 3,
              "initialContribution": 26000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 251000,
              "provider": "halifax",
              "term": 14,
            },
            Object {
              "fixed": 2,
              "id": 4,
              "initialContribution": 27000,
              "monthlyPayment": 1976,
              "mortgageAmount": 233800,
              "mortgageTotal": 257000,
              "provider": "santander",
              "term": 14,
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
